index.js

```diff
@@ -21,6 +23,14 @@ module.exports = function (cmd, args, opts) {
 			return val;
 		};
 
+		if (opts.preferLocal) {
```

Comment:
`if(opts.preferLocal === true)` maybe?

Not sure what you want to do if someone passes in `{preferLocal: 'hello world'}`. The value is truthy, but it's not `true` :).